src/main/resources/plugin.yml

```diff
@@ -1,5 +1,5 @@
 name: TradeShop
-version: 1.6.0-BETA
+version: 1.6.0-Updater
```

Comment:
Copying from above:

> What's this? This isn't a valid release tag. I assume you wanted to check wether the latest build had been compiled...?
If yes, this should be changed back to BETA.

Reply:
Same as above